shared/cluster.go

```diff
@@ -34,52 +35,63 @@ type Pod struct {
 	Node      string
 }
 
-// ManageWorkload creates or deletes a workload based on the action: create or delete.
-func ManageWorkload(action, workload, arch string) (string, error) {
+// ManageWorkload applies or deletes a workload based on the action: apply or delete.
+func ManageWorkload(action string, arch string, workloads ...string) (string, error) {
+	if action != "apply" && action != "delete" {
+		return "", fmt.Errorf("invalid action: %s. Must be 'apply' or 'delete'", action)
+	}
+	
 	var res string
 	var err error
 
-	if action != "create" && action != "delete" {
-		return "", fmt.Errorf("invalid action: %s. Must be 'create' or 'delete'", action)
-	}
-
-	resourceDir := BasePath() + "/distros-test-framework/workloads/amd64/"
-	if arch == "arm64" {
-		resourceDir = BasePath() + "/distros-test-framework/workloads/arm/"
+	resourceDir := BasePath() + "/distros-test-framework/workloads/"
```

Comment:
this path here should have the / amd ,  right ?

Reply:
I can just remove the if below and have default to arch